demo-app/build.gradle.kts

```diff
@@ -10,13 +10,13 @@ plugins {
     alias(libs.plugins.detekt)
 }
 
-fun localProperties(): Properties {
-    val localProperties = Properties()
-    val localPropertiesFile = rootProject.file("local.properties")
-    if (localPropertiesFile.exists()) {
-        localProperties.load(FileInputStream(localPropertiesFile))
-    }
-    return localProperties
+fun secretProperties(): Properties {
+    val properties = Properties()
+    rootProject.file("secrets.properties")
```

Comment:
After merge, the team will have to move their current secrets from `local.properties` to `secrets.properties` or run `bundle exec fastlane run configure_apply` to use the shared secrets.

Reply:
When I try to run this command I get:
>[!] Decryption key could not be found